### DISCUSSION TITLE
SCRAM authentication is not supported by this driver

### DISCUSSION THREAD

**Discussion by sryze** [dbeaver/dbeaver#18335] [Q&A]
When I try to connect to a local PostgreSQL server (v15) via DBeaver I get the same error each time:

```
SCRAM authentication is not supported by this driver. You need JDK >= 8 and pgjdbc >= 42.2.0 (not ".jre" versions)
```

Any idea how to solve it? My installed Postgres driver is up-to-date and I am using JDK 11.

OS: Windows 10 x64
PostgreSQL 15 downloaded from EDB

**Comment by LonwoLonwo** [comment]
Hello.
Maybe this tip can help you: https://stackoverflow.com/a/64546286

**Reply by sryze** [reply]
Thanks, that helped

I actually tried this one time and it didn't work for some reason, so I deleted the PostgreSQL driver then. Now I un-deleted it and updated once again, and it's working now!

**Comment by LonwoLonwo** [comment]
Also, what DBeaver version do you use?

**Reply by sryze** [reply]
22.2.5
